### PR TITLE
Updated set of man pages

### PR DESCRIPTION
**Pull request by fmherschel**
Merged man pages from branch multi-target-lp in fork fmherschel

### CODE REVIEW

**Review by angelabriel — CHANGES_REQUESTED** [review]
@fmherschel  are you sure, that you want to replace all the '.png' files in  SAPHana/fh_test/final_icons/ with a symbolic link to /data/video?
That will break the files. If we do not longer need the '.png' files, we should simply remove them.

**Comment by fmherschel** [comment]
@angelabriel. Oh, oh I did moved the files locally and it was never intended that this changes comes to github.
Now need to check how to fix this one :/

**Comment by fmherschel** [comment]
Could you check, if the files are back as plain PNGs instead of symlinks?

File: SAPHana/ra/SAPHanaTopology [hunk 1]
```diff
@@ -893,6 +893,10 @@ function sht_monitor_clone() {
        vName=${NODENAME}
```
Comment (angelabriel) [inline SAPHana/ra/SAPHanaTopology:893]: In line 380 the changes from your pull request 41 are missing. If we now merge this PR, we will lost the change. Can you please check.

**Review by angelabriel — CHANGES_REQUESTED** [review]
yes, the PNGs are back. Thanks.
But can you please have a look at my comment regarding SAPHanaTopology.

File: SAPHana/ra/SAPHanaTopology (hunk 1 above)
Comment (angelabriel) [reply SAPHana/ra/SAPHanaTopology:893]: No, that was my fault. As we do not have overlapping changes, we will not lost the changes. Sorry for the noise.

**Comment by angelabriel** [comment]
Everything looks good to me. Approving and merging the changes.
Thanks.

**Review by angelabriel — APPROVED** [review]

File: SAPHana/ra/SAPHanaTopology (hunk 1 above)
Comment (fmherschel) [reply SAPHana/ra/SAPHanaTopology:893]: Unfortunately I had the need to use different branches. Will fix that.

**Comment by fmherschel** [comment]
Forget about the last comment as in the meantime you did already approve and merge the PR :)
Have a nice evening, @angelabriel ;-)

**Pull request merged**